qiskit/schemas/qobj_schema.json

```diff
@@ -152,6 +153,24 @@
                     ],
                     "title": "Frame change pulse"
                 },
+                {
+                    "properties": {
+                        "name": {
+                            "enum": [
+                                "parametric_pulse"
+                            ]
+                        },
+                        "pulse_shape": {"type": "string"},
+                        "params": {"type": "array"}
+                    },
+                    "required": [
+                        "pulse_shape",
+                        "params",
+                        "t0",
+                        "ch"
+                    ],
+                    "title": "Frame change pulse"
```

Comment:
Shouldn't this be `"title": "Parametric Pulse"`?

Reply:
wow thank you yes 😱